### ISSUE TITLE
Modified Geant4 example

### ISSUE THREAD

**Issue by gerritzen** [NESTCollaboration/nest#176]
Sorry for the spam.

To get accommodated to NEST, I modified one of the Geant4 examples ([see here](https://github.com/gerritzen/geant4/tree/NEST_example/examples/extended/optical/LXe)). I figured this might be useful for others. If you agree, I can either include the code in here and create a PR, or you can include a link in the README. If not, feel free to ignore this issue.

P.S. I didn't find an elegant way to edit a ready-made physics list, so I copy-pasted G4OpticalPhysics and just changed a few lines.

**Comment by mszydagis** [comment]
@KianTrengove   Can you take a look at this issue?

**Comment by mszydagis** [comment]
Yes, that sounds reasonable to me, @KianTrengove. Could you please work with @gerritzen on getting this done and into NEST? (e.g., a PR for me to approve). And, apologies, @gerritzen for leaving you hanging for over 2 years on this issue (!)
